src/git-unix/ogit-http-fetch/main.ml

```diff
@@ -0,0 +1,169 @@
+module Store = Git_unix.Store
+module Sync = Git_unix.Sync (Store) (Git_curl)
```

Comment:
May be it's time to make a simple `Git_cohttp` with `cohttp-lwt-unix` and use it instead `git-unix.curl`. An example of what we should have is available inside `src/git-mirage/` (but this one use `Cohttp_mirage`).